bbb-graphql-server/metadata/databases/BigBlueButton/tables/public_v_user_voice_activity.yaml

```
@@ -6,6 +6,16 @@ configuration:
   custom_column_names: {}
   custom_name: user_voice_activity
   custom_root_fields: {}
+object_relationships:
+  - name: user
+    using:
+      manual_configuration:
+        column_mapping:
+          userId: userId
```

```suggestion
          meetingId: meetingId
          userId: userId
```

Always use `meetingId + userId` for joining with `user`, because sometimes the `userId` is duplicated to join in a breakout room only in audio!